indexing-service/src/test/java/org/apache/druid/indexing/overlord/TaskQueueTest.java

```diff
@@ -623,6 +640,25 @@ private HttpRemoteTaskRunner createHttpRemoteTaskRunner()
     );
   }
 
+  private static void verifyTaskStatusCounts(final TaskQueue taskQueue, int successCount, int failureCount)
+  {
+    Assert.assertEquals(
+        successCount,
+        taskQueue.getSuccessfulTaskCount().values().stream().mapToLong(Long::longValue).sum()
+    );
+    Assert.assertEquals(
+        failureCount,
+        taskQueue.getFailedTaskCount().values().stream().mapToLong(Long::longValue).sum()
```

Comment:
It's better to keep these two assertions in separate methods as that would make for more readable test cases.